### ISSUE TITLE
Should We LREM the listAckName after send events to channel?

### ISSUE THREAD

**Issue by lovemelovemycode** [fengpeiyuan/flumeng-plugins-redis#2]
com.flumeng.plugins.source.RedisListPollableSource.process() donot LREM the listAckName after send events to channel.Maybe we should LREM the listAckName  after send events to channel successfully.

**Comment by fengpeiyuan** [comment]
yeah, LREM operation happens in sink(https://github.com/fengpeiyuan/flumeng-plugins-redis/blob/master/src/main/java/com/flumeng/plugins/sink/WithRedisListAckSink.java) to ensure events to store.
